seps/1933-workload-identity-federation.md

```diff
@@ -0,0 +1,228 @@
+# SEP-1933: Workload Identity Federation
+
+- **Status**: Draft
+- **Type**: Standards Track
```

Comment:
Also to clarify - this is intended as an _extension_, similar to other things in [`ext-auth`](https://github.com/modelcontextprotocol/ext-auth/tree/main)? Or is the intent for this to be part of the core spec?

Reply:
Correct. This is intended as an extension.